### PR TITLE
chore: update hardcoded colors to use color registry in CustomPick, Image, Preferences, task-manager, and move provider colors to ts file

### PR DESCRIPTION
**Pull request by SoniaSandler**
Signed-off-by: Sonia Sandler <ssandler@redhat.com>

### What does this PR do?
This PR updates `CustomPick`, images, preferences, andtask-manager to use the color registry instead of hardcoded colors (with a small change to the faded background of CustomPick to include background opacity without affecting the other components )
It also includes moving the hardcoded provider colors to a separate .ts file as to not have any hardcoded colors is .svelte files

### Screenshot / video of UI
CustomPick after:
Screenshot from 2024-11-19 17-25-40
Screenshot from 2024-11-19 17-25-28

### What issues does this PR fix or reference?
Related to https://github.com/podman-desktop/podman-desktop/issues/9080

### How to test this PR?

- [ ] Tests are covering the bug fix or the new feature

### CODE REVIEW

File: packages/renderer/src/lib/task-manager/TaskManager.svelte [hunk 1]
```diff
@@ -59,7 +59,7 @@ window.events?.receive('toggle-task-manager', () => {
           <div class="text-xs uppercase ml-2">tasks</div>
           <div class="flex-1"></div>
           <!--
-          <div title="Toggle Do Not Disturb Mode" class="cursor-pointer hover:bg-charcoal-600 p-1">
+          <div title="Toggle Do Not Disturb Mode" class="cursor-pointer p-1">
```
Comment (benoitf) [inline packages/renderer/src/lib/task-manager/TaskManager.svelte]: todo: let's remove the full comment block

**Review by benoitf — APPROVED** [review]

File: packages/renderer/src/lib/ui/ProviderInfoCircle.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
+/**********************************************************************
+ * Copyright (C) 2023-2024 Red Hat, Inc.
```
Comment (benoitf) [inline packages/renderer/src/lib/ui/ProviderInfoCircle.ts]: nitpick
```suggestion
 * Copyright (C) 2024 Red Hat, Inc.
```

File: packages/renderer/src/lib/ui/ProviderInfoCircle.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+// Each provider has a colour associated to it within tailwind, this is a map of those colours.
+// bg-purple-600 = podman
+// bg-sky-300    = docker
+// bg-sky-600    = kubernetes
+// bg-gray-900   = unknown
```
Comment (benoitf) [inline packages/renderer/src/lib/ui/ProviderInfoCircle.ts]: non-blocking: AFAIK you could remove this comment as it's the same definition few lines after

**Review by benoitf — APPROVED** [review]
thanks for the changes

**Review by feloy — APPROVED** [review]

**Pull request merged**